src/main/java/projectLibraryPartOne/service/LibraryServiceImpl.java

```diff
@@ -0,0 +1,91 @@
+package projectLibraryPartOne.service;
+
+import projectLibraryPartOne.entity.Book;
+
+import java.util.ArrayList;
+
+public class LibraryServiceImpl implements LibraryService {
+
+    private ArrayList<Book> listBooks;
```

Comment:
we work at the interface level (List< Book>)

Reply:
fixed

Reply:
not see. You need write:
private List< Book> listBooks;